### PR TITLE
Components

### PR DESCRIPTION
**Pull request by RLewis627**
Added all react components (src/components)
Added user-defined CSS (src/css)
Deleted items that are no longer needed (logos, icons, react-native CSS)
Added build folder with contents of `npm build`
Changed App.tsx to accommodate a multi-page web app with HashRouter

### CODE REVIEW

File: src/components/Thanks.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import '../css/ThankYouPage.css';
+
+class Thanks extends React.Component {
+  render() {
+    return <p>Thank you for using Therabot!</p>;
+  }
+}
```
Comment (Nicholas-Baron) [inline src/components/Thanks.js]: Could this be rewritten as a function?
```
export default function Thanks(){
  return ... ;
}
```

File: src/components/UserLoginBox.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      this.ChkSignIn = this.ChkSignIn.bind(this);
+  }
+
+  nextPath(path) {
+    this.props.history.push(path);
+  }
+
+  ChkSignIn() {
+      var LoginUseName = document.getElementById("LoginUseName").value;
+      var Loginpin = document.getElementById("LoginPIN").value;
```
Comment (Nicholas-Baron) [inline src/components/UserLoginBox.js]: Could these be `let` or even `const`?

File: src/components/Register.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    nextPath(path) {
+        this.props.history.push(path);
+    }
+
+    ChkSuccessReg() {
+        var FirstName = document.getElementById("FirstName").value;
+        var UseName = document.getElementById("UseName").value;
+        var EmailAdd = document.getElementById("EmailAdd").value;
+        var pin = document.getElementById("pin").value;
```
Comment (Nicholas-Baron) [inline src/components/Register.js]: Could these be `let` or `const`?

File: src/components/Conversation.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        super(props);
+        this.AppendUserConvo = this.AppendUserConvo.bind(this);
+    }
+
+    nextPath(path) {
+        this.props.history.push(path);
+    }
+
+    AppendUserConvo() {
+        var UserComment = document.getElementById("ConvoUserInput").value;
```
Comment (Nicholas-Baron) [inline src/components/Conversation.js]: Could this be `let` or `const`?

File: package.json [hunk 5]
```diff
@@ -11,11 +11,17 @@
...
+    "@types/react-router-dom": "^5.1.4",
     "eslint-plugin-flowtype": "^3.13.0",
     "gh-pages": "^2.2.0",
+    "jquery": "^3.5.0",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
+    "react-router": "^5.1.2",
+    "react-router-dom": "^5.1.2",
     "react-scripts": "3.3.1",
+    "reactstrap": "^8.4.1",
```
Comment (Nicholas-Baron) [inline package.json]: I'll allow it, but please limit the usage of this package.

**Review by Nicholas-Baron — CHANGES_REQUESTED** [review]
I strongly recommend removing the build folder from this commit. It is unnecessary and just bloats the repository.
When the pull request merges, I will begin writing the tests in Typescript and will convert the files to Typescript.

**Comment by RLewis627** [comment]
- Also removed build folder
- Attempted to remove jquery dependency as well, but converting the 2 Jquery functions that exist in Conversation.js to their Javascript counterparts resolves to various errors in the console.
- Attempted to convert files to tsx but it resolves to a large amount of errors that I don't really know how to fix.

File: src/components/Conversation.js [hunk 6]
```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import '../css/ConversationStyle.css';
+import '../css/ThankYouPage.css';
+import $ from 'jquery';
+//import firebase from "../firebase.js"
+
+class Conversation extends React.Component {
+    constructor(props) {
+        super(props);
```
Comment (Nicholas-Baron) [inline src/components/Conversation.js:9]: The props are unused. I will rewrite the class as a functional component.

**Review by Nicholas-Baron — APPROVED** [review]
I will begin rewriting the class style components into function style components and adding tests.

**Pull request merged**